packages/react-tabster/src/hooks/useTabster.ts

```diff
@@ -0,0 +1,20 @@
+import { useFluent } from '@fluentui/react-shared-contexts';
+import { getCurrentTabster, createTabster } from 'tabster';
+
+/**
+ * Tries to get a tabster instance on the current window or creates a new one
+ * @internal
+ * @returns Tabster core instance
+ */
+export const useTabster = () => {
```

Comment:
```suggestion
export const useTabster = (): TYPE | null => {
```

_nit_ can you please return types there explicitly